src/integrationtest/java/com.microsoft.aad.msal4j/DeviceCodeIT.java

```diff
@@ -123,6 +123,15 @@ private void runAutomatedDeviceCodeFlow(DeviceCode deviceCode, User user){
             } else {
                 SeleniumExtensions.performADLogin(seleniumDriver, user);
             }
+
+            if (environment.equals("azurecloud") && !isADFS2019) {
```

Comment:
Consider adding to constants file

Reply:
In the latest commit, this value now comes from the constants in AzureEnvironment.java